frontend/src/lib/components/CompactList/CompactList.scss

```diff
@@ -33,6 +33,8 @@
     }
 
     .secondary-text {
-        color: var(--text-secondary);
+        .posthog-3000 & {
+            color: var(--text-secondary);
+        }
     }
```

Comment:
I'd be tempted to just 

```
.posthog-3000 .secondary-text {
    color: var(--text-secondary);
}
```

I had to google to remind myself what the trailing ampersand does 🙈 

But I'm not a yardstick of what good CSS is :)

Reply:
Yep, that's what it does (lets you declare a top-level parent selector within nested scss without having to create a new top-level rule for it)! We also use this for other 3000 styles, but fortunately once we completely migrate over, we'll be able to simplify all of this!